### DISCUSSION TITLE
Xls instructions file

### DISCUSSION THREAD

**Discussion by seattlebuster** [gpstar81/GPStar-proton-pack#97] [Q&A] [answered]
I built my first pack using the .xls instruction file.  I seemed to have deleted it and noticed it is no longer in the download.  Does anyone still have it that could re-upload it?  I'm getting ready to build my 84 pack

**Comment by gpstar81** [comment, marked as answer]
The old xls spreadsheet can be found in the old commits way back in February.

Here is a direct link to the file to download it:

https://github.com/gpstar81/haslab-proton-pack/blob/c8a6a58fb3dfb822e205b915b4e95d157d04c9bd/Instructions.xls
